phpstan.neon.dist

```diff
@@ -304,7 +304,7 @@ parameters:
         - '~^Call to an undefined method atk4\\ui\\JsChain\:\:emit\(\)\.$~'
         - '~^Call to an undefined method atk4\\ui\\JsChain\:\:clearData\(\)\.$~'
         - '~^Call to an undefined method atk4\\ui\\JsChain\:\:addJsonData\(\)\.$~'
-        - '~^Access to an undefined property atk4\\ui\\UserAction\\JsExecutorInterface&atk4\\ui\\View\:\:\$viewForUrl\.$~'
+        # - '~^Access to an undefined property atk4\\ui\\UserAction\\JsExecutorInterface&atk4\\ui\\View\:\:\$viewForUrl\.$~'
```

Comment:
completely delete it (exclude rules in "generated" section), it is actually the state we want to achive

Reply:
I was testing to see if this was the case.

Reply:
👍